yamcs-client/src/main/java/org/yamcs/client/utils/WellKnownTypes.java

```diff
@@ -51,10 +48,23 @@ public static ListValue toListValue(Object[] array) {
         return listb.build();
     }
 
+    public static String toStringValue(byte[] array) {
```

Comment:
We have a function in StringConverters that does that.

Reply:
I tried that initially, importing StringConverters in a yamcs-client class was not possible after discussing with Fabian.

Reply:
Yes, true, but please check if we don't have that function somewhere else in the yamcs-client.